lib/rucio/core/transfer.py

```diff
@@ -441,10 +441,20 @@ def set_transfers_state(transfers, state, submitted_at, external_host, external_
             if rowcount == 0:
                 raise RucioException("%s: failed to set transfer state: request doesn't exist or is not in SUBMITTING state" % rws)
 
+            stmt = select(
+                models.DataIdentifier.datatype
+            ).where(
+                models.DataIdentifier.scope == rws.scope,
+                models.DataIdentifier.name == rws.name,
+            )
+            datatype = session.execute(stmt).scalar_one_or_none()
+
             msg = {'request-id': rws.request_id,
                    'request-type': rws.request_type,
                    'scope': rws.scope.external,
                    'name': rws.name,
+                   'dataset': None,
+                   'datasetScope': None,
```

Comment:
should these two be `dataset-name` and `dataset-scope` instead?

Reply:
If we follow the ~~technical debt~~ convention used in other parts of rucio, it must the way I put it. See here for example: https://github.com/rucio/rucio/blob/2a41a64899f41b356660fff80a7530765b5d59e0/lib/rucio/core/trace.py#L174

Reply:
ouch